docs/tutorial/image_cls.md

```diff
@@ -106,15 +106,18 @@ If configuration finishes, the configuration file is generated in the `cifar10_t
 Train your model by running `blueoil train` with model configuration.
 
     $ docker run --rm \
-        -e CUDA_VISIBLE_DEVICES=0 \
+        --gpus device=0 \
```

Comment:
In my environment (Docker v19.03.7), this doesn't work. 
I think we need to use `--gpus '"device=0"'`. It's not beautiful though... 😭 
See: https://github.com/NVIDIA/nvidia-docker/issues/1257

Reply:
It works without the `'` or `"` quotation mark. I've checked. Should I include them anyway?

Reply:
Done